app/feed/page.tsx

```diff
@@ -1,8 +1,11 @@
 import { PostEditor } from '@/components/post-editor/PostEditor';
 import PostFeed from '@/components/post-feed/PostFeed';
 import WelcomeTexts from '@/components/welcome-texts/WelcomeTexts';
+import { getMumblePosts } from '@/utils/helpers/posts/getMumblePosts';
 
+// TODO this page can be deleted. as index is the same- check
```

Comment:
I added a task for removing all TODOs in the code before we submit the project: https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/issues/17